tests/integration/test_cli.py

```diff
@@ -210,27 +210,6 @@ def test_scripts(PipenvInstance):
         assert 'which python' in c.stdout
 
 
-@pytest.mark.cli
-def test_scripts_resolve_dot_env_vars(PipenvInstance):
```

Comment:
I wrote this test recently but technically put it in the wrong file -- relocated it in this PR and extended it's use case.